### DISCUSSION TITLE
💸 Free to Play [EXPERIMENT]

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4308] [Ideas]
Free to Play games have the biggest market for casual gaming and have the potential to help us scale to more users, especially on the mobile market.

The goal of free to play at Sunflower Land is to retain more incoming users & increase revenue. 

### Pay Wall Problem

The game currently implements a soft pay wall, where players must purchase BBs if they want to restock their shops & keep playing. This is a hurdle and confusing for new players who are often blocked to progress through the game or they decide not to play - overall this leads to churn inside of our game. 😢 

### Free to Play Experiment

To overcome this problem, we propose a Free to Play version of the game where players can get limited stock each day for free. This will enable them to perform basic daily actions and become more immersed into the game.

This new system will become available to only **10% of new players** & will stay in **experiment** stage for a while.

The restock option provides roughly 35% of BB usage inside of the game - we need to tweak and ensure the new system can help fund the project’s growth & does not effect the economy.

This is an experiment, which means if it does not have the intended outcome it will be rolled back.

## Solution

In the new model, we will provide a free refill of stock every **24 hours**. This will be conveyed as a ship which comes with new stock. This provides them with basic stock to do farming operations. The shipment will appear near the dock and the player can claim it for **free**.

The stock size will be 30% the amount that is currently available. If a player depletes their stock, they are able to use a BB to call a new shipment in and give a full restock (same as current size)

**Revenue generation**

There are 2 components to our **new** revenue model:

1. Higher retained players = more revenue opportunities
2. Reoccurring optional purchases.

City Builders using the free to play model typically focus on hard currencies to perform 'instant actions' such as instant building/harvests etc. Players can use the hard currency to skip wait times.

We will experiment with the same options in Sunflower Land. Players can optionally use a BB to:

- Instant build
- Instant expand
- Instant cook

These will be expensive and not suitable for long term, but still a good use case for short term gameplay, onboarding advancing & reoccurring revenue generation.

The hard currency required (Block Bucks) would be based on the time for the action to complete. For instance a 1 minute action would take 0.1 BB whereas a 30 min action could take 1 BB.

**Block Bucks > Gems**

The team have previously mentioned migrating to a higher denomination for our hard currency at a ratio of 1:100. This would allow us to offer different hard currency pricing on certain actions (instead of fractionalisation of BBs)

### Anti-bots

Free to Play games reduce the risk for botters or malicious actors to try extract from the game. A new system will need to tighten the grip on how these accounts could impact on the economy.

There are multiple suggestions we will implement on the security side. However, we will also perform game facing tweaks to ensure the system is not advantageous to bots:

1. Lvl 20 required to sell on P2P
2. Lvl 40 required to withdraw crops/resources
3. Hard currency (block buck) is required to sell on P2P (if you don't have VIP access).

The aim is to plug up areas that can be easy extractions for F2P extractors.

## Challenges

We have identified a few core challenges and issues with a free to play system that we are iterating ideas on.

**Value pegging**: In our current system players must use BBs to unlock seeds and harvest crops. This creates an interesting dynamic where $ input is required to produce a resource. We will need to ensure the amount of seed stock & trading limits are suitable so free to players do not hinder market values. 

**Bots**: A free to play system is more enticing for botters to abuse. Our P2P trading system was previously a target for bots to trade crops to earn SFL and will continue to be in this new system. In a free to play scenario we need to ensure the F2P model cannot impact the economy. A suggestion include removing the 1 free trade limit and replace it with 1 BB per trade (if you don't have a VIP pass). Some variability will also be considered for more complex levelling loops to disrupt any botting strategies

**Speed boost competition**: The ability to use Block Bucks to instant cook does compete with Cooking Boost SFTs (e.g. Luna's Hat). However, we believe this can be a positive for these NFTs as it can help drive perceived value towards these boosts. (Should I spend $4 worth of BBs a day on instant cooking, or buy the Lunas Hat one-off).

## Conclusion

We would love some feedback on this experiment! We will try multiple different ideas and take a data focused approach to ensure our changes do not impact economy or revenue.

As always we would love your ideas and collaboration on this proposal. Thanks! 🤗

**Comment by elowine** [comment]
Hello, thank you for the new information about the free-to-play system,

Have you looked at NFT's rental option to go from F2P to P2E?

For example, a free rental by the renter, but the renter must pay a % of the resource generated / or a paid rental in a coin or sfl?

**Reply by adamhannigan** [reply]
It is a great suggestion, something the community definitely wants 💯 

On our end it is more of an issue that our legal team need to address. The rental market is a bit of a grey area since it can turn a game item into an asset which generates a predictable yield in the form of a cryptocurrency - this could fall into a financial asset category. Since there hasn't been any legal defining on this area we are treading carefully.

We will get some legal clarification on this first and share the results 👍 The option you mentioned of renting for coins would circumvent these problems 👌

**Comment by Thebyber** [comment]
I like the proposal for free to play but... the main problem that I see for a free to play player is the cost of obtaining resources such as an iron pickaxe, getting sand shovels for the new mechanics, being able to fish with multiple rods.. 

So I would add a calendar with daily rewards, these rewards can be taken if the player takes the daily chest or even without having to take the daily chest... I leave it to your choice.

Here I leave an example: 

Day 1: 5 axes
Day 2: 400 sunflower seeds 
Day 3: 2 squid
Day 4: 5 fishing rods with 5 earthworm
Day 5: 200 potato seeds
Day 6: 1 sunflower cake
:
:
Day 14: 1 BB
:
:
Day 28: 1 BB

The idea would be to offer the following every 3 days: 
-materials to carry out actions on the island: cutting down trees, mining, fishing, etc.
-seeds to plant when running out of stock
-fish or foods so they can level up.

And give away 1 BB every 14 days so they can restock:

If one day they do not come to play, they will be one day behind on the calendar.

If a month has more than 28 days, the same reward could be given in the last 3 days.

**Reply by adamhannigan** [reply]
Thanks @Thebyber 👌 This is a nice alternative approach.

Giving these rewards we can predict what sort of activity a player needs to be doing. It could be used in conjunction with a lower stock amount for players.

**Comment by Amber2234** [comment]
Sorry, this is going to be long. I’ve been playing for about a month and I’m at Level 23, so I believe I bring in some valuable experience. In real life, I’ve successfully run a German company for over 20 years, so I also bring some economic insights to the table.
Let me first say that I really appreciate playing your game and I think you’ve already made many good decisions. As u know a Free-to-Play model is both fascinating and risky. Despite some good points, I still don't find the proposal 100% satisfactory. Here are a few criticisms:
1. "This new system will become available to only 10% of new players and will stay in the experimental stage for a while." This sentence alone renders the system absurd. Who are the lucky 10% of players? Will I need to start 10 farms just to be lucky enough to be included? A time-limited experiment is fine. You can’t take away their accounts afterward. You have to care tha game takes no harm and it shouldn’t be a marketing disaster.
2. "In the new model, we will provide a free refill of stock every 24 hours." This is way too much. I’ve been running my farm for 18 or 19 days (my second attempt at a farm). Around level 16, I had to buy BBs for the first time to restock. I bought 20 BBs, and I still have 17 left. The game is fairly balanced at the beginning stage, including the use of BBs. Just keep it that way, and it’s better to give new players a few BBs to keep them playing. It’s okay if it’s a bit challenging to get a BB. 
3. Instant build, instant expand, instant… I slightly disagree, though I understand the point. Selling BBs to let players speed things up is absolutely fine. IMHO Sunflower Land already fulfills the basics of attracting players. In my first attempt, I logged in with Google data, played for around 8 days, and quickly advanced to about level 12 (first buildings built almost instantly…). The progression speed felt pretty good. Unfortunately, it then became cumbersome with the sequence wallet (gas fee…) and I restarted by logging in with my metamask wallet. 2nd time the progression speed still felt pretty good.

Don’t get me wrong. F2P is the way to go to let Sunflower Land prosper. Despite 3 years of experience with Axie Infinity and 5 years of crypto experience with ETH, starting to play SFL was still quite clumsy. What are the real hurdles normal people face when beginning the game?
1. Logging into the game with my Google account seemed absolutely perfect to me. THIS is the way to attract new players. Also, allowing iCloud, Facebook, etc. would be even better. The first problems definitely come with the wallets. I think it was with the Daily Chest. Until then, I was absolutely happy. But, of course, I wanted the chest and followed the recommendation in the documentation to use the Sequence Wallet. But that wasn’t cool. 
2. Putting MATIC on the Polygon network is still a mess. If I didn’t already have some crypto resources, I probably would have given up again. Why? I wanted to buy Polygon with Apple Pay. To pay $10, I would have had to give Unlimited (?) my full name, address, etc., just to get some “evil” crypto. Finally, I just bridged some ETH into MATIC using my Metamask wallet directly. The not-so-small fee annoyed me, but by that point, I didn’t care anymore. Overall, it wasn’t cool. 
3. The last hurdle is definitely around level 10-15 when you slowly realize that you inevitably have to spend money to restock. As a new player, I was immediately suspicious. You can’t estimate how much money you’ll need in the long run. And you wonder: Why? In “normal” games, you can also play without spending money. Spending money is usually optional (Thx to Supercell, etc.), which isn’t a problem (Thx to the whales). It would be almost unique for SFL to be a F2P game in the crypto world…

So, my proposal and last words:
1. Block Bucks are already your „gems“. A new 4th or 5th “gem” currency is IMHO a No-Go.
2. SFL has the possibility to attract players into the crypto world. It must feel like a standard iPad game like Sim City or the like. F2P at the base, giving plenty of opportunities to spend money which is already 90% in place. I have no idea what is technically possible. Handle small amounts of SFL without a wallet, or provide the gas fee for small players yourself. The main thing is that the need to deal with wallets and gas fees should disappear initially (for the first 20+ levels?).
3. The rental model was proposed here. Although I was initially positive about it, I’m now unsure if it’s the right model. Does it attract the right players or just some click bots? You can provide limited BBs for player acquisition. All commercially successful games give out some “gems,” but not enough to “play at the top.” A new NPC could give quests to deliver crops and pay with 1 BB. The crops are burned to avoid impacting the rest of the economy. Optionally (as a second step), high-level players could also set these tasks for new players to earn BBs: produce 250 Sunflowers and 120 Potatoes for 1 BB, etc. This way, you’d also have a new earning opportunity for older players and a better “Rental” model but you’lll have economic effects.

 Finally, thanks for this great game. Keep it up. ’m hooked and will stick with it. 
Regards, Amber aka 225729

**Reply by adamhannigan** [reply]
Thanks @Amber2234 for the in-depth response. You've put a lot of thought into this and I appreciate it 🤗 

I will clarify a few things.

The experiment is **time-limited** and is used as an A/B testing strategy. We have dozens of changes & improvements that go live every week which makes it hard to know which change lead to positive results. We have previously ran these experiments on onboarding changes and had great insights (often leads us to not build a feature). These experiments run for 2-3 weeks and then we take them out.

We understand players have already spent a lot of BBs on restocks and there is also the challenge of pegging real world value to stocks. For the initial version we aim to give the bare minimum stock to player (20-30% of what it currently is). In reality, if you want to be advancing or attaining SFTs you will still need to use BBs. The free stock will only help give the newer players some momentum - once they have advanced lvl 15+ we anticipate they will purchase BB to get the advantage they need (bonus stock each day)

Your experience with onboarding was great to hear - this is the sort of engagement we are after. There are other areas with wallets/technical/onramping that we will also improve outside of this proposal. We do have significant data that highlights a large churn rate from expansions 5-9, which is Day 1-4. When we measure this experiment we will be using this in success metrics. Our hypothesis is that especially during this time the F2P model will help retain players.

You had some great points are abstracting wallets + gas fees from the initial parts of the game. This is definitely aligned with the team's thinking about upcoming improvements.

I'm not sure if I understood the part around Block Bucks versus Gems. Gems will replace Block Bucks in the game so we only have 1 hard currency. The aim is to migrate (for free instantly) your BBs to gems at a 1:100 ratio. Having a larger denomination will let us use different pricing for different actions. Currently 1 BB is the cheapest unit (~0.15 USD), we may have actions that need to be a little bit cheaper or more expense (i.e. 66 Gems ~0.10 USD or 130 Gems ~0.20).

Thanks!

**Reply by Amber2234** [reply]
Sorry for the confusion. I have no criticism from my side regarding a simple "fractionalization" of the Block Bucks. I would prefer a 1:10 split, otherwise bids in auctions will soon be in the millions of Bucks.

We shouldn’t mix too many things. With over 20 years of business experience, I can assure you that your growth curve is absolutely healthy. The daily players peaked at 17.8k and are now consolidating around 8k. This is totally fine. The next expansion phase will surely come, and it usually has little to do with game content and 90% to do with marketing.

Personally, I find it a great idea to move closer to a free-to-play (F2P) model with the options discussed. But going F2P alone probably won’t bring players. It’s mostly marketing actions that attract players. That’s why it’s good to then have reduced the churn rate and lowered the barriers to playing.

Reducing churn rate: From my own in-game experience, you could do this by significantly increasing the game content. At levels 5-9, the tasks get repetitive for the first time. It needs stories for the NPCs (their history, their connections to each other, daily mails, world background). You need quests like “bring this here,” “fetch that,” etc., just to keep the game from getting boring. Important: At around level 9, it becomes really frustrating to constantly get delivery tasks above your level or ones that are nearly impossible to complete without spending a considerable amount of money. The game content becomes “thin,” and you start feeling the bland mechanics. My advice as a next step would be to make it a smooth, fun experience up to level 20. This includes improving the beginner’s guide in the boat under the island. This should reduce the churn rate, and it should be measurable.

Attracting players: Maybe it’s not that complicated? What’s the current marketing strategy? Things like Twitter are the absolute minimum and should be standard practice. Changing things in-game probably won’t increase the player base in a meaningful way. The simplest and strongest method is word of mouth. A community coin is really not enough for this. It would be better to reward players with SFL (or USDC) and/or valuable in-game bonuses like extra crops, treasure keys, and the like, so that satisfied players will tell their friends.

What you probably need are measurable marketing actions. Making news would be my choice (see below), as it’s one of the cheapest actions. Have you considered traditional advertising or partnering with YouTube influencers? As you mentioned, measuring the impact is crucial, and that’s easily trackable through links and in-game codes. Even if you develop a mobile version and a more user-friendly, non-crypto-like game experience (as we discussed), you will still eventually have to pay for advertising. And I can assure you, the day will come when you’ll be more than happy to simply spend money to acquire new players, as other actions are far more tedious. Making news is cheap: Remember Farmers World on the WAX chain? Yes, a bot-driven game, but the first of its kind and with a large following. How about generously but temporarily accepting their resources and offering a smooth entry into Sunflower Land, allowing players to comfortably progress to level 15? This could attract some cheap attention from potential players.

One last thing:There’s another hidden hurdle causing mid-level player churn. It’s one thing that it’s hard to play the game without spending money, but there’s also a feeling that the game doesn’t reward you fairly. This becomes very obvious with the delivery tasks. Often, they provide less reward than you would earn by just selling the ingredients at the market. So, you’re essentially losing money just to play for fun and complete your delivery tasks. The YouTuber "Casual Gamerz" might even stop making SFL videos, which is really unfortunate. He points out that the typical SFL player, despite a lot of effort and overcoming many hurdles, can’t earn money. He’s not entirely wrong. As the SFL coin is already becoming too deflationary, maybe it’s a good idea to be a little more generous.

**Reply by adamhannigan** [reply]
Thanks for the ideas 🤗 Some good examples for improving the gameplay for levels 1-20 👌 

For a bit more context, part of the driver for this initiative is to amplify the marketing streams. We've done a lot of KOLs, paid ads, sponsorships + more that work but are not profitable due to the churn rates with onboarding players. The F2P aspect aims to increase the retention so these marketing avenues can be profitable.

There definitely are lots of ways to increase engagement and we will be exploring + implementing a bunch of these ideas 🙏 Cheers!

**Comment by alexandrelira99** [comment]
Wouldn't be better to wait for mobile launch for those free to play experiments? Talking about my scenario, i can bring literally 10 new players if the game is on play/apple stores.

**Comment by Dionis404** [comment]
Adam, consider the idea of making resupply for performing certain actions. 
Conditional weekly\Daily tasks that the player will be able to restock by completing. 
The tasks should be as simple as possible and achievable by all players. 
For example, talk to Pete in the square. 
Harvest any crop. 
Cook a dish. 
Go to the location. 

It is also worth considering that such a top-up would give points as in the idea (https://github.com/sunflower-land/sunflower-land/discussions/3773). 

I'm really looking forward to the moment when the game becomes more accessible to Web2 users. 


UPD: It's worth paying attention to this: https://github.com/sunflower-land/sunflower-land/discussions/4467

**Comment by bichobravo** [comment]
There is a problem with cheap, fast-growing crops. It's extremely annoying to keep clicking when there are 400 seeds, you could have a mini crop machine or some globim that could pay with coins to plant and harvest one crop at a time or a longer time for cultivation, or each island has a less efficient crop machine corresponding one powered by sunflower oil or something like that produced by the person himself. lateral development that does not depend on evolution with different strategies would be very interesting.

**Reply by lordleejin** [reply]
When you expand to the Desert Island there is a Crop Machine you can build powered by Oil that you will be mining. Basic crops Sunflower, Potato and Pumpkin can be planted into it. I believe the proposal for Skill revamp will add more crops as well. 👻♥️🚀

**Reply by Amber2234** [reply]
Personally I like the way of harvesting found in many other games. One click on a crop, a scythe appears and then sweep over adjacent crops to harvest them all.

**Reply by adamhannigan** [reply]
I hear you, I'd like to explore a nicer gesture for planting + harvesting 💯
